src/index.html

```diff
@@ -21,7 +24,413 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
```

Comment:
The 'is-active' class suggests that this is a modifier in BEM methodology, but it's not following the BEM naming convention. It should be 'nav__link--is-active' if 'nav__link' is the block element.